src/components/BaseLayout.tsx

```diff
@@ -22,10 +22,12 @@ export const BaseLayout = ({ children, locale }: Props) => {
   return (
     <html lang={locale}>
       <body
-        className={`antialiased ${roboto.className} flex min-h-screen flex-col`}
+        className={`antialiased ${roboto.className} flex min-h-screen flex-col bg-[#f8f8f8]`}
```

Comment:
Maybe we should start using Tailwind variables instead of hardcoding colors, so that it's easier to change them later